almond/data/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bashio
+WAIT_PIDS=()
+TODAY=$(date +%s)
+
+ha_config=$(\
+    bashio::var.json \
+        host "$(hostname)" \
+        port "3001" \
+)
+
+almond_config=$(\
+    bashio::var.json \
+        kind "io.home-assistant" \
+        hassUrl "http://hassio/homeassistant" \
+        accessToken "${HASSIO_TOKEN}" \
+        refreshToken "${HASSIO_TOKEN}" \
+        accessTokenExpires "$((TODAY + 62208000))" \
```

Comment:
How far ahead in the future is this? 

This value is in ms. 

This is what I have in the integration

```python
                    # 5 years from now in ms.
                    "accessTokenExpires": time.time() + (60 * 60 * 24 * 365 * 5) * 1000,
```